practice_problems_3/signup.php

```diff
@@ -0,0 +1,109 @@
+<?php
+if ($_SERVER["REQUEST_METHOD"] === "POST") {
+    if (empty($_POST["name"])) {
+        echo "Name is required";
```

Comment:
This is not the ideal way of sending a response to an API call. Use http_response_code and json data here